src/service/git/git-cli.ts

```diff
@@ -42,12 +42,14 @@ export class GitCLIService {
    * @param branch branch which should be cloned
    */
   async clone(from: string, to: string, branch: string) {
-    if (!fs.existsSync(to)) {
-      // don't use this.git since it will configure git with local user.name and user.email which requires cwd to be a git repo
-      await simpleGit().clone(from, to, ["--quiet", "--shallow-submodules", "--no-tags", "--branch", branch]);
-    } else {
-      this.logger.warn(`Folder ${to} already exist. Won't clone`);
+    if (fs.existsSync(to)) {
+      this.logger.warn(`Folder ${to} already exist. Deleting and recreating it`);
+      fs.rmSync(to, { recursive: true, force: true });
+      fs.mkdirSync(to, {recursive: true});
```

Comment:
@lampajr I was about to merge this when I realized we don't need `mkdirSync` since git clone will create the destination folder no? wdyt?

Reply:
actually you are right, I do not remember why I explicitly created it.. btw fixing it